packages/ws-server/src/webSocketServer.ts

```diff
@@ -213,6 +215,10 @@ app.ws.use(async (ctx) => {
 
         ctx.websocket.send(JSON.stringify(response));
     });
+
+    setInterval(async () => {
```

Comment:
Q: How does affect subscriptions with messages?
Do they receive empty messages not conforming with log formats?

Reply:
Messages related to subscriptions need to have a `subscriptionId` property and messages that are in repsonse to a called api method have a `method` property. The ping message doesn't have either of those, so it doesn't get handled by the clients.